Sources/ProjectDescription/GraphDefinition.swift

```diff
@@ -0,0 +1,71 @@
+import Foundation
+
+/// A styling definition to be used when elements are rendered in graphs
+public struct GraphDefinition: Hashable, Equatable, CustomStringConvertible, CustomDebugStringConvertible, Codable {
```

Comment:
Maybe `GraphAppearance`?

Reply:
I knew I had to ask before implementing 😅 
Will try to refactor it everywhere